src/pages/instances/InstanceList.tsx

```diff
@@ -354,61 +360,23 @@ const InstanceList: FC = () => {
               </>
             )}
             {hasInstances && selectedNames.length === 0 && (
-              <>
-                <SearchBox
-                  className="search-box margin-right u-no-margin--bottom"
-                  name="search-instance"
-                  type="text"
-                  onChange={(value) => {
-                    setQuery(value);
-                  }}
-                  placeholder="Search"
-                  value={query}
-                  aria-label="Search"
-                />
-                <Select
-                  className="u-no-margin--bottom"
-                  wrapperClassName="margin-right filter-state"
-                  onChange={(v) => {
-                    setStatus(v.target.value);
-                  }}
-                  options={[
-                    {
-                      label: "All statuses",
-                      value: "any",
-                    },
-                    ...instanceStatuses,
-                  ]}
-                  value={status}
-                  aria-label="Filter status"
-                />
-                <Select
-                  className="u-no-margin--bottom"
-                  wrapperClassName="margin-right filter-type"
-                  onChange={(v) => {
-                    setType(v.target.value);
-                  }}
-                  options={[
-                    {
-                      label: "Containers and VMs",
-                      value: "any",
-                    },
-                    ...instanceListTypes,
-                  ]}
-                  value={type}
-                  aria-label="Filter type"
-                />
-              </>
+              <InstanceSearchFilter
+                instances={instances}
+                filters={filters}
+                setFilters={setFilters}
```

Comment:
The component keeps its state when we switch project, we can avoid that by using it as a key: `key={project}`, hence forcing a clean render of it.